### ISSUE TITLE
SC problematic for closed - 1.4.10 Reflow: Clarification needed for scrollable content areas

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#394]
Opening this issue to work on a tweak in language to the 1.4.10 Reflow note in the SC Problematic for Closed Functionality section per the following comment on issue 221.
 
> @maryjom I'm not saying it should block publication.  I would be in favor or wording that allowed for any mechanism which might allow for adjusting the scrolling content at the specified width/height (which may include resizing the viewport or content).
>_Originally posted by @mraccess77 in https://github.com/w3c/wcag2ict/issues/221#issuecomment-2181392785_

The reasoning given was a viewport could apply to any resizable control or area within a window (e.g. a resizable input field, listbox, etc.) and not just a resizable window. 

We may want to clarify that point in the SC Problematic for Closed Functionality section. We should also check to see if the clarification is also needed in the section Applying SC 1.4.10 Reflow to Non-web Documents and Software.

**Comment by maryjom** [comment]
This issue will be addressed after the publication of the public draft.

**Comment by WayneEDick** [comment]
Dear Mary
Perhaps it is time to remind people why 1.4.10 was added in the first
place. I have attached a paper on the reasoning underlying reflow. The
operational overhead caused by 2-dimensional scrolling is approximately the
number of lines times the number of screens required to hole a full line
without reflow.  For 125 words that is about 50 scrolls. See Chart 1 in the
article. It tells the story. When we talk about blocks of text, 125 words
is already significant. 100 lines is a huge deal.
One thing I did not mention was the source of the 200% enlargement
guideline for readability. It is too small for almost everyone with
impaired visual acuity. It is a legacy of print on paper. In that
environment 200% is the largest print size that could be implemented. 400%
would turn a pamphlet into a phonebook.
Remember this article. Any medium that displays print for reading without
reflow is excluding almost everyone from reading.
A lot is made of apps and other ICT that does not involve a lot of writing,
but the truth is that  many apps use a lot of print. If they don't reflow,
most people with low vision can't read.
When I entered college there was no accomodation. There was reader service
but my campus posted this service in small print on a crowded bulletin
board. I flunked out. When I returned with accommodation, I graduated with
high honors and completed my PhD in mathematics. Also, take that into
account the numbers for operational overhead were computed by a
mathematician.
Best, Wayne

**Comment by mitchellevan** [comment]
Relevant public WCAG discussion: https://github.com/w3c/wcag/issues/3322

**Comment by mitchellevan** [comment]
> Dear Mary Perhaps it is time to remind people why 1.4.10 was added in the first place. I have attached a paper on the reasoning underlying reflow.

Hi @WayneEDick , I don't see the attachment

**Comment by maryjom** [comment]
@mitchellevan I don't intend to make any further changes prior to publication of the wide review draft. The 1.4.10 Reflow guidance already points to the Understanding document's Intent section which provides necessary background on the requirement. However, WCAG2ICT must rely upon the normative requirement. This issue is talking about one Note, out of context of the full guidance in the WCAG2ICT draft. I opened the issue so we didn't lose the comment made by Jon Avila, which brings up a valid point.

**Comment by WayneEDick** [comment]
I am resending my paper with a link to my Google Drive.
The bottom line is that enlargement without reflow kills reading
comprehension. The reader can read letters, but has profound difficulty
understanding paragraphs. An informational medium that uses 100 words or
more has this problem. Viewport access that requires 2-dimensional
scrolling is inaccessible. If a medium enables reflow on platforms A, B and
C, and fails on E, F and G, is accessible on A, B, C and inaccessible on E,
G and G. That is reality.

https://docs.google.com/document/d/15ElCfY9VWzhPm3OSNofBe2x5Kt-nzt-TQFnZTcThV8o/edit?usp=sharing

Best, Wayne

**Comment by bruce-usab** [comment]
@WayneEDick and @mraccess77 there a short window of opportunity to add to the WCAG2ICT note.

**Comment by bruce-usab** [comment]
Discussed on call 8/1.

**Comment by maryjom** [comment]
The task force has made clarifications to the SC Problematic for Closed Functionality and Applying SC 1.4.10 Reflow to Non-web Documents and Software sections. In both cases we have added text to make it clear this SC applies to scrollable elements as well. See PR #500 and PR #505 for the exact changes made. You can read the changes in-context in the editor’s draft in [Applying SC 1.4.10 Reflow to Non-web Documents and Software](https://w3c.github.io/wcag2ict/#applying-sc-1-4-10-reflow-to-non-web-documents-and-software) (Note 5) and [SC Problematic for Closed Functionality](https://w3c.github.io/wcag2ict/#success-criteria-problematic-for-closed-functionality) (the bullet for 1.4.10 Reflow).

@mraccess Please review to ensure these improvements address your concern.

**Comment by WayneEDick** [comment]
Hi Bruce,
Do you want me to study and comment on an issue?
Best Wayne

**Comment by maryjom** [comment]
Closing as completed.
